src/AnalysisServices/AnalysisServices.Test/InMemoryTests/DataPlaneCommandTests.cs

```diff
@@ -61,7 +61,7 @@ public class DataPlaneCommandTests : AsTestsBase
                                         + "65kxhZWVUbTHaPuEvg03ZQ3esDb6wxQewJPAL-GARg6S9wIN776Esw8-53AWhzFu0fIut-9FXGma6jV7"
                                         + "MYPoUUcFuQzLZgphecPyMPXSVhummVCdBwX9sizxnmFA";
 
-        public DataPlaneCommandTests(ITestOutputHelper output)
+        public DataPlaneCommandTests(ITestOutputHelper output) : base(output)
         {
             XunitTracingInterceptor.AddToContext(new XunitTracingInterceptor(output));
```

Comment:
Remove this line. It's removed in other test classes, and doing so will not fail any tests.